src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,72 @@
  */
 
 public class Plane {
+    private String model;
+    private int countOfSeats;
+    private double price;
+    private int buildYear;
+    private String color;
 
+    public Plane(PlaneBuilder builder) {
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public int getCountOfSeats() {
+        return countOfSeats;
+    }
+
+    public double getPrice() {
+        return price;
+    }
+
+    public int getBuildYear() {
+        return buildYear;
+    }
+
+    public String getColor() {
+        return color;
+    }
+
+    public static class PlaneBuilder {
+        private String model;
+        private int countOfSeats;
+        private double price;
+        private int buildYear;
+        private String color;
+
+        public PlaneBuilder() {
+        }
```

Comment:
do you need to explicitly declare this empty constructor?

Reply:
done